app/src/main/java/org/mozilla/fenix/compose/TabSubtitleWithInterdot.kt

```diff
@@ -47,9 +48,18 @@ fun TabSubtitleWithInterdot(
 
     Layout(
         content = {
-            TabSubtitle(text = firstText)
-            TabSubtitle(text = " \u00b7 ")
-            TabSubtitle(text = secondText)
+            SecondaryText(
+                text = firstText,
+                fontSize = 12.sp
+            )
+            SecondaryText(
+                text = " \u00b7 ",
+                fontSize = 12.sp
+            )
+            SecondaryText(
+                text = secondText,
+                fontSize = 12.sp
+            )
```

Comment:
```suggestion
            SecondaryText(
                text = firstText + " \u00b7 " + secondText,
                fontSize = 12.sp
            )
```

Reply:
We're actually doing some measurements in the preceding code to ensure all the text are visible so it actually has to be separated into 3 SecondaryText.